### DISCUSSION TITLE
The user enrolment count in dashboard of course is mismatching.

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Lern/Community#163] [Issues]
Hi team,
We are using Sunbird version 4.6.0. We are facing the issue with the Enrolment user count in the Dashboard. The count shown on the dashboard is not matching with count in the exhaust reports. We don't see any issues in the Druid and all the tasks are running fine in it.Kindly let us know why there can be the count mismatch.

cc: @raghupathiguduri

**Comment by kumarks1122** [comment]
@Venkatesh-Dayani23 Exhaust report jobs extracts enrolment details from report cluster where the user_enrolments table data will present with altered schema. This report cluster data is getting updated by cassndra migratory job.
Can you please check if the Cassandra migrator job is scheduled and running successfully?

**Comment by Venkatesh-Dayani23** [comment]
Hi @kumarks1122 
We are not running the Cassandra Migration Job as we are not using the Report Cassandra.In our case we are able to get the enrolment count from both exhaust reports and dashboard but the count is not matching. Enrolment count got from the reports is higher than the enrolment count shown in the dashboard.Kindly need your suggestion on this.

**Reply by kumarks1122** [reply]
@Venkatesh-Dayani23 Enrolment count for the dashboard is taken from Druid for the dashboard. Can you schedule a call to check this?

**Reply by kumarks1122** [reply]
cc: @AmiableAnil

**Comment by Venkatesh-Dayani23** [comment]
@kumarks1122  Kindly let us know your availability to schedule a call tomorrow.

cc: @raghupathiguduri

**Reply by kumarks1122** [reply]
@Venkatesh-Dayani23 can you schedule the call for 2 PM?

**Reply by Venkatesh-Dayani23** [reply]
sure @kumarks1122  we can have the call at 2 PM

cc: @raghupathiguduri

**Reply by Venkatesh-Dayani23** [reply]
@kumarks1122  can we connect now ?

**Reply by kumarks1122** [reply]
@Venkatesh-Dayani23 Yes

**Reply by Jayaprakash8887** [reply]
@Venkatesh-Dayani23 Kindly let us know if the query is resolved. Please mark the thread as answered.

**Comment by AmiableAnil** [comment]
Closing this thread. Please feel free to reopen if required.
